content/guides/dashboard/runs.md

```diff
@@ -77,6 +77,17 @@ spec. You can also download the video.
 
 <DocsImage src="/img/dashboard/videos-of-recorded-test-run.png" alt="Video of test runs" ></DocsImage>
 
+### Cypress App
+
+For even more visibility into your test results, information about recent runs
+is surfaced in the [Cypress App](/guides/core-concepts/cypress-app). Aggregate
```

Comment:
@mattvCypress should this be "Cypress app"?

Reply:
@pstakoun The link should be "Cypress app". The header appears to be title cased so "Cypress App" (row 80) is technically fine there.